### PR TITLE
fix: animate resize to prevent flickers

### PR DESCRIPTION
**Pull request by KaustubhKumar05**
### Steps to repro:

- Open chat/participant list when multiple tiles are present
- The issue is more noticeable when downsizing the screen

### Bundle size increase:

Prod gzip:
<br />
image
<br />
Deployment gzip:
<br />
image

### CODE REVIEW

File: packages/roomkit-react/src/VideoTile/StyledVideoTile.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 export const Root = styled('div', {
   padding: '0.75rem',
-  // show videotile context menu on hover
-  // [`&:hover .tile-menu`]: {
-  //   display: 'inline-block',
-  // },
 });
 
+const MotionRoot = motion(Root);
+MotionRoot.defaultProps = { layout: true };
```
Comment (raviteja83) [inline packages/roomkit-react/src/VideoTile/StyledVideoTile.tsx:11]: can we try to do this without a library? already the roomkit-react size is huge.

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/VideoTile/StyledVideoTile.tsx:11]: Using the library helps with more coverage and better performance than hardcoding transforms: https://www.framer.com/motion/examples/#layout-animations

Loom:
https://www.loom.com/share/5e9f099f435d4f72a519f4ae5f698132?sid=4d313667-7fc9-489d-8c95-06b27ed703b9

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/VideoTile/StyledVideoTile.tsx:11]: The lib supports tree-shaking

Comment (raviteja83) [reply packages/roomkit-react/src/VideoTile/StyledVideoTile.tsx:11]: can we check what is the difference that it is making so it works without much changes?

**Comment by ygit** [comment]
@KaustubhKumar05 can you resolve the conflicts on this PR?

**Review by ygit — APPROVED** [review]
LGTM

**Review by hdz-666 — APPROVED** [review]

**Pull request merged**